File: fitness-tracker/src/app/model/WorkoutDTOs/CreateWorkoutCoachDTO.ts
import { CreateExerciseSetDTO } from '../ExerciseSetDTOs/CreateExerciseSetDTO';

export interface CreateWorkoutCoachDTO {
  userId: number;
  name: string;
  comment: string;
  scheduled: string;
  createWorkoutSetDTOList: Array<CreateExerciseSetDTO>;
}

File: fitness-tracker-backend/src/main/java/tech/fitnesstackerbackend/fitnesstrackerbackend/model/workout/dtos/CreateWorkoutCoachDTO.java
package tech.fitnesstackerbackend.fitnesstrackerbackend.model.workout.dtos;

import lombok.Getter;
import tech.fitnesstackerbackend.fitnesstrackerbackend.model.exerciseset.CreateExerciseSetDTO;

import java.util.List;


@Getter
public class CreateWorkoutCoachDTO {
    private Integer userId;
    private String name;
    private String comment;
    private String scheduled;
    private List<CreateExerciseSetDTO> createWorkoutSetDTOList;
}
